### PR TITLE
Added disable-artifact flag.

### PR DESCRIPTION
**Pull request by elad-pticha**

### CODE REVIEW

File: action.yaml [hunk 1]
```diff
@@ -61,6 +61,10 @@ inputs:
     description: Fail the CI if Cimon encountered an error
     required: false
     default: "false"
+  upload-artifact:
```
Comment (alex-ilgayev) [inline action.yaml:64]: Can you rename it to `report-artifact-log` and put it after `report-process-tree` ?

File: src/main/index.js [hunk 2]
```diff
@@ -72,6 +74,7 @@ async function run(config) {
         '--volume', '/sys/kernel/debug:/sys/kernel/debug:ro',
         '--volume', '/home/runner/work:/github_workspace',
         '--env', `CIMON_LOG_LEVEL=${config.cimon.logLevel}`,
+        '--env', `CIMON_UPLOAD_ARTIFACT=${config.cimon.uploadArtifact}`,
```
Comment (alex-ilgayev) [inline src/main/index.js:77]: This flag isn't supported yet in Cimon. We can add it later on (on another PR) once we add it to Cimon

File: src/main/index.js [hunk 3]
```diff
@@ -15,6 +15,7 @@ function getActionConfig() {
     const preventionMode = core.getBooleanInput('prevent');
     const allowedIPs = core.getInput('allowed-ips');
     const allowedHosts = core.getInput('allowed-hosts');
+    const uploadArtifact = core.getBooleanInput('upload-artifact');
```
Comment (alex-ilgayev) [inline src/main/index.js:18]: See comment below

File: src/main/index.js [hunk 4]
```diff
@@ -44,6 +45,7 @@ function getActionConfig() {
             clientId: clientId,
             secret: secret,
             featureGates: featureGates,
+            uploadArtifact: uploadArtifact,
```
Comment (alex-ilgayev) [inline src/main/index.js:48]: See comment below

**Review by alex-ilgayev — CHANGES_REQUESTED** [review]
See my comments. In addition we should update the README with the new flag :)

File: src/main/index.js (hunk 2 above)
Comment (elad-pticha) [reply src/main/index.js]: Do you think we should remove it for now?

File: README.md [hunk 5]
```diff
@@ -33,6 +33,7 @@ The action supports the following parameters:
 | `github-token`           | `${{ github.token }}`               | GitHub token (used to overcome GitHub rate limiting)                            |
 | `report-job-summary`     | `true`                              | Report results through job summary output                                       |
 | `report-process-tree`    | `false`                             | Enable to report the process tree                                               |
+| `report-artifact-log`    | `true`                              | Report results through artifact                                                 |
```
Comment (alex-ilgayev) [inline README.md:36]:
```suggestion
| `report-artifact-log`    | `true`                              | Report logs through job artifact                                                |
```

**Review by alex-ilgayev — APPROVED** [review]
Additional minor change, and you can merge

**Pull request merged**